libsolutil/Common.h

```diff
@@ -73,6 +73,9 @@ using StringMap = std::map<std::string, std::string>;
 // String types.
 using strings = std::vector<std::string>;
 
+/// Max depth a recursion should go before erroring out
+const size_t MAX_RECURSION_DEPTH = 256;
```

Comment:
We have other numbers in other places (for example ParserBase::increaseRecursionDepth)

Reply:
I did this as a reaction to @cameel's feedback.. I can remove it again if you think we don't need it..

Reply:
Oh, you mean that that number should also be replaced by this one?

Reply:
At the very least the comment is wrong I guess

Reply:
I think that @chriseth meant that `increaseRecursionDepth()` uses a different value. It's 1200 there.

If there's no special reason behind 256 other than it being a round number, we could use the same number in both.